### ISSUE TITLE
[BUG] Unix.Packed.Coinminer-6856324-0 FOUND

### ISSUE THREAD

**Issue by zhengr** [cnrancher/autok3s#456]
**Describe the bug**
After clamav scan, in the log I can see:

/usr/local/bin/autok3s: Unix.Packed.Coinminer-6856324-0 FOUND

**Comment by orangedeng** [comment]
What's the autok3s version? You can do `autok3s version` and get the version reply.

**Comment by ly5156** [comment]
@zhengr 
What's the autok3s version and which binary package?
`autok3s_darwin_amd64`, `autok3s_darwin_arm64`, `autok3s_linux_amd64`, `autok3s_linux_arm`, or `autok3s_linux_arm64`, ....

**Comment by ly5156** [comment]
I reproduced:
autok3s_linux_amd64 [v0.4.9](https://github.com/cnrancher/autok3s/releases/tag/v0.4.9)

```shell
clamscan autok3s_linux_amd64 
```
Output：
```
Loading:     8s, ETA:   0s [========================>]    8.62M/8.62M sigs      
Compiling:   2s, ETA:   0s [========================>]       41/41 tasks 

/Users/yanliu/Downloads/autok3s_linux_amd64: Unix.Packed.Coinminer-6856324-0 FOUND

----------- SCAN SUMMARY -----------
Known viruses: 8619000
Engine version: 0.105.0
Scanned directories: 0
Scanned files: 1
Infected files: 1
Data scanned: 26.36 MB
Data read: 24.75 MB (ratio 1.06:1)
Time: 10.571 sec (0 m 10 s)
Start Date: 2022:06:21 08:20:21
End Date:   2022:06:21 08:20:32
```

**Comment by ly5156** [comment]
@zhengr 
1. This issue will be fixed in `v0.5.1`, if you want to use it earlier, try `v0.5.1-rc2`
1. This issue is caused by Upx compression

**Comment by zhengr** [comment]
Thanks for quick fix.
